docs/yul.rst

```diff
@@ -546,11 +546,18 @@ as explained below) and all declarations
 introduce new identifiers into these scopes.
 
 Identifiers are visible in
-the block they are defined in (including all sub-nodes and sub-blocks).
+the block they are defined in (including all sub-nodes and sub-blocks),
+functions also before their declaration but variables only starting from the statement after
+the ``VariableDeclaration``.
```

Comment:
```suggestion
the block they are defined in (including all sub-nodes and sub-blocks).
Variables are only visible from the statement after their declaration (using ``let varName := ...``), whereas functions are also visible before their declaration.
```

Reply:
I thought about that, but if we add a full stop there, the first sentence contracdicts the second.

Reply:
I'm okay with keep it together, but the line

> functions also before their declaration but variables

doesn't sound right.